tests/test_helpers/helpers.go

```diff
@@ -17,16 +17,18 @@ import (
 
 // TmpDir will be created inside this directory
 const testParentDir = "/tmp/gocryptfs-test-parent"
+
+// GocryptfsBinary is the assumed path to the gocryptfs build.
 const GocryptfsBinary = "../../gocryptfs"
 
-// "go test" runs package tests in parallel! We create a unique TmpDir in
-// init() so the tests do not interfere.
+// TmpDir is used by "go test" running package test s in parallel! We create a
+// unique TmpDir in init() so the tests do not interfere.
```

Comment:
TmpDir is a unique temporary directory. "go test" runs package tests in parallel. We create a unique TmpDir in init() so the tests do not interfere.